mdsa/algorithms/svd.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import scipy
+
+from mdsa.algorithm import Algorithm
+
+"""
+ Performs svd using a LAPACK (Linear Algebra Package) routine
+"""
+
+
+class Svd(Algorithm):
```

Comment:
I think it would look better too, but I didn't do it because I assumed the flake8 linting would disagree:
https://www.python.org/dev/peps/pep-0008/#class-names

What conventions have we followed in scikit-bio, qiita, etc. for these class names? Strict PEP8 or OK to do `SVD` ?